docker-for-windows/release-notes.md

```diff
@@ -13,6 +13,42 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
+## Docker Desktop 3.6.0
+2021-08-10
+
+### New
+
+- You can now create a Dev Environment from your local git repository
+- Add sorting to volume list
+- Add name search to volumes list
+
+### Upgrades
+
+- [Compose V2 RC1](https://github.com/docker/compose-cli/releases/tag/v2.0.0-rc.1)
+- Go runtime to 1.16.7
+- Docker Engine to v20.10.8
+- containerd to v1.4.9
+- runc to v1.0.1
+- Kubernetes to v1.21.3
+- Golang to 1.16.6
+
+### Bug fixes and minor changes
+
+- Fixed bug where the DNS server would fail after receiving an unexpectedly large datagram.
+- Fixed spurious traces on iptables updates.
+- Fixed slowness when adding multiple ports forwarding option.
+- Fixed bug where the WSL 2 sync code creates dangling symlinks if the WSL 2 home directory is the same as the Windows home directory. Fixed [docker/for-win#11668](https://github.
+  com/docker/for-win/issues/11668).
+- Fixed `docker context ls` after upgrade from 3.5.x when the Linux WSL 2 home directory is the same as the Windows home directory.
+- Fixed the permissions on `%PROGRAMDATA%\Docker` to avoid a potential Windows containers compromise. See https://github.com/moby/moby/security/advisories/GHSA-8465-jm73-xmj9.
```

Comment:
The URL gives a 404. Could you please check?

Reply:
I'll remove that.